main/dconf/APKBUILD

```
@@ -1,32 +1,28 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dconf
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
 arch="all"
 license="LGPL-2.1"
```

```suggestion
license="LGPL-2.1-or-later"
```
https://wiki.gnome.org/action/show/Projects/dconf#dconf_License